Fix instruction when app is missing enableFlutterDriverExtension

sethladd opened the pull request:
This is a stop-gap fix. We plan for a more robust fix later.

In the meantime, closes https://github.com/flutter/flutter/issues/3171

sethladd committed:
Fix instruction when app is missing enableFlutterDriverExtension

This is a stop-gap fix. We plan for a more robust fix later.

In the meantime, closes https://github.com/flutter/flutter/issues/3171

Update driver.dart

sethladd committed:
Fix instruction when app is missing enableFlutterDriverExtension

This is a stop-gap fix. We plan for a more robust fix later.

In the meantime, closes https://github.com/flutter/flutter/issues/3171

sethladd commented:
cc @yjbanov

abarth committed:
Dismissable doesn't resize from its full size (#3194)

Previously we'd snap to the size of the background widget before resizing to

zero. Now we force the background widget to match the size of the dimissable

widget at the time it was dismissed.



Fixes #3092

devoncarew committed:
docs cleanup (#3196)

* some docs cleanup

* update styles

* remove unused file

jason-simmons committed:
Some documentation for running "flutter refresh" in the Gradle sample (#3201)

Commit:
Added Scrollbar, removed ScrollbarPainter (#3203)

* Added Scrollbar, removed ScrollbarPainter

* removed a dead import

* updated per review feedback

* Only call dispatch if the widgets State is still mounted

yjbanov committed:
[driver] fix analyzer warnings in the extension (#3205)

yjbanov commented on packages/flutter_driver/lib/src/driver.dart:
```diff
@@ -118,8 +118,9 @@ class FlutterDriver {
           'Timed out waiting for Flutter Driver extension to become available. '
-          'To enable the driver extension call registerFlutterDriverExtension '
-          'first thing in the main method of your application.'
+          'Ensure your application (lib/main.dart) imports '
```
More precisely: `'Ensure your test app imports '`

A test app is not necessarily `lib/main.dart`

sethladd replied:
Thanks. Done.

yjbanov commented:
LGTM

abarth committed:
Run `pub get` if flutter_tools.stamp changes (#3206)

This will help folks run pub get after they upgrade their flutter.git
revision.

abarth committed:
Add `flutter precache` (#3207)

This command explicitly populates the flutter tool's cache of binary artifacts.
Also, teach `flutter create` to update the cache in case its the first command
that a user runs.

jason-simmons committed:
Do not check for an existing package map if a command is going to run "pub get" to update the package map (#3202)

Also remove a redundant check for the --pub option in the run command

abarth committed:
Remove pub package version skew (#3212)

We need to pin the version of package:analyzer we use to avoid version skew
within our project.

devoncarew committed:
move services to using cache.dart (#3211)

vlidholt committed:
Fixes spelling in gallery button demo (#3209)

devoncarew committed:
have flutter precache print if there's no work to do (#3213)

* have flutter precache print if there's no work to do

* tweak precache

abarth committed:
Update engine (#3214)

Also, update hello_android to use the new Java class names.

abarth committed:
Convert some Columns into Blocks (#3210)

These columns were secretly re-creating Block in a more complex way. Now we
just use Block directly.

vlidholt committed:
Moves and renames gallery demos (#3217)

abarth committed:
Add more dartdoc to material.dart (#3204)

The dartdoc will continue until morale improves.

abarth committed:
Improve the docs for Overlay (#3216)

Fixes #3197

abarth committed:
FlexibleSpaceBar fails to animate (#3222)

We need to listen to the Scaffold's animation so that we rebuild every time it
ticks so that we can drive our animations.

devoncarew committed:
rev the dart sdk to 1.16.0-dev.4.0 (#3221)

devoncarew committed:
add a target for android-x64 (#3224)

* add a target for android-x64

* update armeabi-v7a to x86_64

Commit:
Added OverscrollIndicator, removed OverscrollIndicatorPainter (#3220)

* Added OverscrollIndicator, removed OverscrollIndicatorPainter

vlidholt committed:
Updates selection controls gallery demo (#3225)

Commit:
Scrollbar cleanups (#3229)

Commit:
Remove ScrollableListPainter (#3226)

* Remove ScrollableListPainter

stevemessick committed:
Add validity checks to create (#3215)

* Add validity checks to create

* Adjust indent

krisgiesing committed:
Update wording on viewport test (#3228)

devoncarew committed:
fix an issue parsing adb devices (#3235)

sethladd committed:
Update driver.dart

devoncarew committed:
Fix windows crash (#3236)

* better messaging about windows support

* fix lints

sethladd committed:
Merge branch 'sethladd-patch-2' of github.com:flutter/flutter into sethladd-patch-2

sethladd commented:
whoa. too many commits. closing.
